app/components/modals/WithdrawModal.tsx

```diff
@@ -252,6 +240,35 @@ export default function WithdrawModal({
             </div>
             <p className="text-foreground font-semibold">Withdrawal Successful!</p>
             <p className="text-sm text-foreground/70">Your funds are on the way.</p>
+            
+            {txid && (
+              <div className="w-full">
+                <h3 className="text-sm font-medium text-accent-2 mb-2 text-center">Transaction ID</h3>
+                <a
+                  href={`https://mempool.space/tx/${txid}`}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  className="block bg-secondary p-3 border border-border hover:border-accent-3 transition-colors group"
+                >
+                  <p className="text-foreground break-all text-sm font-mono group-hover:text-accent-3 transition-colors">
+                    {txid}
+                  </p>
+                  <p className="text-xs text-foreground/50 mt-1 group-hover:text-accent-3/70 transition-colors">
+                    Click to view on mempool.space →
+                  </p>
+                </a>
+              </div>
+            )}
```

Comment:
Small suggestion, here, the txid link will display a 64-character hex string. We can use `aria-label` will allow better accessibility for screen readers.

```suggestion
                <a
                  href={`https://mempool.space/tx/${txid}`}
                  aria-label="View transaction on mempool.space"
                  target="_blank"
                  rel="noopener noreferrer"
                  className="block bg-secondary p-3 border border-border hover:border-accent-3 transition-colors group"
                >
                  <p className="text-foreground break-all text-sm font-mono group-hover:text-accent-3 transition-colors">
                    {txid}
                  </p>
                  <p className="text-xs text-foreground/50 mt-1 group-hover:text-accent-3/70 transition-colors">
                    Click to view on mempool.space →
                  </p>
                </a>
              </div>
            )}
```